website/content/en/docs/deployment/vanilla/guide-terraform.md

```diff
@@ -17,15 +17,39 @@ Specifially, you must:
 
 ## Deployment Steps
 
+### Directory
```

Comment:
For consistency can we make this just as bullet point as part of pre-requisities?

Reply:
done